deploy/gppkg/madlib.spec.in

```diff
@@ -7,7 +7,7 @@
 BuildRoot:      @MADLIB_GPPKG_RPM_SOURCE_DIR@
 Summary:        Apache MADlib for @GPDB_VARIANT@ Database
 License:        @CPACK_RPM_PACKAGE_LICENSE@
-Name:           Apache_MADlib
+Name:           madlib
```

Comment:
The change from `madlib` to `Apache_MADlib` created some problems during upgrade tests so I decided to revert it. The rpm will still be renamed to `apache-madlib-1.12-bin-Linux.rpm`